contracts/hardhat.config.js

```diff
@@ -415,51 +415,59 @@ module.exports = {
   etherscan: {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY,
-      arbitrumOne: process.env.ARBISCAN_API_KEY,
+      arbitrumOne: process.env.ETHERSCAN_API_KEY,
       holesky: process.env.ETHERSCAN_API_KEY,
-      base: process.env.BASESCAN_API_KEY,
-      sonic: process.env.SONICSCAN_API_KEY,
-      hoodi: process.env.HOODISCAN_API_KEY,
+      base: process.env.ETHERSCAN_API_KEY,
+      sonic: process.env.ETHERSCAN_API_KEY,
+      hoodi: process.env.ETHERSCAN_API_KEY,
       plume: "empty", // this works for: npx hardhat verify...
     },
     customChains: [
+      {
+        network: "mainnet",
+        chainId: 1,
+        urls: {
+          apiURL: "https://api.etherscan.io/v2/api?chainId=1",
+          browserURL: "https://etherscan.io",
+        },
+      },
       {
         network: "holesky",
         chainId: 17000,
         urls: {
-          apiURL: "https://api-holesky.etherscan.io/api",
+          apiURL: "https://api.etherscan.io/v2/api?chainId=17000",
           browserURL: "https://holesky.etherscan.io",
         },
       },
       {
         network: "base",
         chainId: 8453,
         urls: {
-          apiURL: "https://api.basescan.org/api",
+          apiURL: "https://api.etherscan.io/v2/api?chainId=8453",
           browserURL: "https://basescan.org",
         },
       },
       {
         network: "sonic",
         chainId: 146,
         urls: {
-          apiURL: "https://api.sonicscan.org/api",
+          apiURL: "https://api.etherscan.io/v2/api?chainId=146",
           browserURL: "https://sonicscan.org",
         },
       },
       {
         network: "plume",
         chainId: 98866,
         urls: {
-          apiURL: "https://explorer.plume.org/api",
+          apiURL: "https://api.etherscan.io/v2/api?chainId=98866",
```

Comment:
I don't think Plume (and some of the other chains) will work on this etherscan endpoint. Can you point me to the docs where you found this?